cypress/integration/Editor/slate_block_picker_spec.js

```diff
@@ -21,11 +21,6 @@ describe("can enter both element types SlateBlockPicker and SlateVisualElementPi
     cy.get('[cy="slate-block-picker-menu"]').should('be.visible');
   });
 
-  afterEach( () => {
```

Comment:
Hvorfor feiler dette igjen? 
Knappen forsvinner jo når jeg trykker utenfor editoren?

Reply:
Forsvinner for oss ja, men det er ikke alltid at z-indexen i css'en blir lik -1, altså det afterEach() var bygd rundt. Det hendte at alle testene passa og når jeg kjørte det for andre gang så tryna de på 4 test, eller på 1 test, altså knappen forsvant men z-indexen var fortsatt 1. Helt random hvordan oppførselen er pga. den async update funksjonen og awaiten på innsiden med 50ms sleep.